js/src/product-feed/issues-table-card/issues-table-data.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * External dependencies
+ */
+import { EmptyTable, Table } from '@woocommerce/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import WarningIcon from '.~/components/warning-icon';
+import ErrorIcon from '.~/components/error-icon';
+import AppDocumentationLink from '.~/components/app-documentation-link';
+import EditProductLink from '.~/components/edit-product-link';
+
+/**
+ * The rows with data for the Issues table
+ *
+ * @param {Object} args The data and the headers
+ * @param {Object} args.data The data containing the issues to render.
+ * @param {Object[]} args.headers An array with the different headers
+ * @return {JSX.Element} The rendered component with the issues or with an empty table if no data is provided
+ */
+const IssuesTableData = ( { data, headers } ) => {
+	if ( ! data ) {
+		return (
+			<EmptyTable headers={ headers } numberOfRows={ 1 }>
+				{ __(
+					'An error occurred while retrieving issues. Please try again later.',
+					'google-listings-and-ads'
+				) }
+			</EmptyTable>
+		);
+	}
+
+	return (
+		<Table
+			caption={ __( 'Issues to resolve', 'google-listings-and-ads' ) }
+			headers={ headers }
+			rows={ data.issues.map( ( el ) => [
+				{
+					display:
+						el.severity === 'warning' ? (
+							<WarningIcon />
+						) : (
+							<ErrorIcon />
+						),
+				},
+				{ display: el.product },
+				{ display: el.issue },
+				{
+					display: (
+						<AppDocumentationLink
+							context="issues-to-resolve"
+							linkId={ el.code }
+							href={ el.action_url }
+						>
+							{ el.action }
+						</AppDocumentationLink>
+					),
+				},
+				{
+					display: el.type === 'product' && (
```

Comment:
💅 
It could be replaced with constant `ISSUE_TYPE_PRODUCT` as well.

Reply:
Good suggestion. 

Changed here -> https://github.com/woocommerce/google-listings-and-ads/pull/1176/commits/1d8aaa69176739180b99b5f886fc5abf58e51978